pipeline/create_pipeline.go

```diff
@@ -99,4 +99,11 @@ func CreatePipeline(p *gaia.CreatePipeline) {
 		gaia.Cfg.Logger.Error("cannot put create pipeline into store", "error", err.Error())
 		return
 	}
+
+	if !gaia.Cfg.Poll {
+		// TODO: Add repo VCS type information?
+		// TODO: Q: Also, save the webhook so we can later find the pipeline associated with it?
+		// A: Don't think so, because the hook will contain the name of the repository.
+		subscribeRepoWebhook(&p.Pipeline.Repo)
```

Comment:
@michelvocks This also means that webhook automatic building will never be available for drop-in pipelines as this needs repository information. Are you okay with this? Or shall we try and do some workaround magic?

Reply:
No, this is absolutely fine! If you just drop your pipeline binary into the pipeline folder, this pipeline cannot use the poll functionality as well as the webhook. 👍